crates/teloxide-core/src/types/chat_permissions.rs

```diff
@@ -156,7 +231,22 @@ struct ChatPermissionsRaw {
     can_send_messages: bool,
 
     #[serde(default, skip_serializing_if = "Not::not")]
-    can_send_media_messages: bool,
```

Comment:
Can you add this back, for backwards compatibility, actually?

I imagine a case where `teloxide` users saved `ChatPermissions` on disk/in a db, without this field after deserializing it back part of the information will be lost.

I imagine something like

```rust
// ...
    can_manage_topics: bool,

    /// Back compatibility hack to correctly deserialize json produced before permission split in TBA 6.5
    #[serde(default, skip_serializing)]
    can_send_media_messages: bool,
}
```
```rust
// ...
            this |= Self::MANAGE_TOPICS
        }

        // Back compatibility hack, see field docs
        if can_send_media_messages {
            this |= Self::SEND_MEDIA_MESSAGES;
        }
```

Reply:
Actually, don't do this. This apparently would break deserialization of json from telegram, since it has `can_send_media_messages` even if _only some_ of the media permissions are there...